packages/outputarea/style/index.css

```diff
@@ -48,6 +49,39 @@
   flex-shrink: 1;
   height: auto;
   overflow-x: auto;
+  user-select: text;
+  -moz-user-select: text;
+  -webkit-user-select: text;
+  -ms-user-select: text;
+}
+
+
+/* pre */
+
+.jp-OutputArea-output pre {
+  border: none;
+  margin: 0px;
+  padding: 0px;
+  overflow-x: auto;
+  overflow-y: auto;
+  word-break: break-all;
+  word-wrap: break-word;
+  white-space: pre-wrap;
+}
+
+
+/* html */
+
+.jp-OutputArea-executeResult.jp-RenderedHTMLCommon {
```

Comment:
The new CSS class arrangement does end up with these two classes on the same element. I have confirmed this can be deleted.